stm32-data-gen/src/rcc.rs

```diff
@@ -250,7 +245,8 @@ impl ParsedRccs {
                             },
                             reset,
                             bus_clock: clock,
-                            stop_mode,
+                            // The default mode is overridden in `low_power.rs`
+                            stop_mode: StopMode::default(),
```

Comment:
instead of leaving it at default here and editing it later, you could do the regexmap lookup here.

(if you need the chip name, just pass it in)

Reply:
Do I have the full chip name here?
I moved it outside, because it was just wb instead of wb55.
Maybe I misunderstood but I think the whole name of the chip is not there yet when parsing the registers

Reply:
oh right this runs *before*. okay let's keep it this way then.